lib/macho/macho_file.rb

```diff
@@ -64,6 +64,10 @@ def magic64?
       Utils.magic64?(header.magic)
     end
 
+    def cmd_round
```

Comment:
Sounds good! I've just been going with `cmd_round` and `round` because of the `round` function in libmacho. `alignment` is much more descriptive.